toolkit/tools/pkg/osmodifierlib/modifierutils.go

```diff
@@ -61,6 +61,67 @@ func doModifications(baseConfigPath string, osConfig *osmodifierapi.OS) error {
 		}
 	}
 
+	if osConfig.Verity != nil {
+
+		bootCustomizer, err := imagecustomizerlib.NewBootCustomizer(dummyChroot)
```

Comment:
The `NewBootCustomizer` and `WriteToFile` only need to be called once and can be shared with both `Verity` and `RootDevice`.

Reply:
updated in pr in new repo